trace/exemplar.go

```diff
@@ -18,14 +18,14 @@ import (
 	"context"
 	"encoding/hex"
 
-	"go.opencensus.io/exemplar"
+	"go.opencensus.io/metric/metricdata"
 )
 
 func init() {
-	exemplar.RegisterAttachmentExtractor(attachSpanContext)
+	metricdata.RegisterAttachmentExtractor(attachSpanContext)
```

Comment:
We need to cleanup these things as well probably. Don't really understand why we need this.

Reply:
Agree `trace` or `tag` shouldn't depend on `exemplar`. I removed `AttachmentExtractor` for now.